storageservice/modules/files/Module.md

```diff
@@ -1,89 +1,39 @@
-
-[Azure File Storage Service](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-introduction)is a shared network file storage service that provides administrators a way to access native SMB file shares in the cloud
-Files stored in Azure File service shares are also accessible via REST APIs. 
-
-# Connector Overview
-
-Azure Storage File Service Connector is used to connect to Azure Storage File Service via Ballerina language easily. It is capable to connect to Azure Storage File Service and to execute operations like getFileList, getDirectoryList, createDirectory, directUpload and getFile etc using the FileClient. It is also capable of executing management operations such as createShare and deleteShare etc using the ManagementClient.
-
-![image](docs/images/AzureFileServiceConnectorOverviewImage.png)
-
-This connector will invoke the REST APIs exposed via the Azure Storage File Service. https://docs.microsoft.com/en-us/rest/api/storageservices/file-service-rest-api
-
-For the version 0.1.0 of this connector, version 2019-12-12 of Azure File Storage Service REST API is used.
-
-# Prerequisites
-
-* Azure Account to Access Azure Portal https://docs.microsoft.com/en-us/learn/modules/create-an-azure-account/
-
-* Azure Storage Account https://docs.microsoft.com/en-us/learn/modules/create-azure-storage-account/
-
-* Java 11 Installed
-Java Development Kit (JDK) with version 11 is required.
-
-* Ballerina SL Alpha 5 Installed
-Ballerina Swan Lake Alpha 5 is required. 
-
-* Shared Access Signature (SAS) or One of the Access Keys for authentication. 
-
-
-## Supported Versions
-
-|                      |  Version           |
-|----------------------|------------------- |
-| Ballerina            | Swan Lake Alpha 5  |
-| Azure Storage Service|     2019-12-12     |
-
-# Quickstart(s)
-
-## Simple operations in Azure File Service File Client.
-These are the simplest scenarios in Azure File Service File Client. You must have the following prerequisites in order 
-to obtain these configurations.
-
-* Azure Account to Access Azure Portal https://docs.microsoft.com/en-us/learn/modules/create-an-azure-account/
-
-* Azure Storage Account https://docs.microsoft.com/en-us/learn/modules/create-azure-storage-account/
-
-* You need to get a Shared Access Key or one of the Access Keys from the Azure Portal.
-
-
-## Step1: Import the Azure Storage Blobs Ballerina Library
+### Step1: Import connector
 
 First, import the `ballerinax/azure_storage_service.files` module into the Ballerina project. 
 
 ```ballerina
     import ballerinax/azure_storage_service.files as azure_files;
 ```
 
-## Step2: Create Azure File Service Configuration
+### Step2: Create a new connector instance
 
-Create the connection configuration using the Shared Access Signature or Access Key, base URL and account name.
+Create a `azure_files:AzureFileServiceConfiguration` with the obtained Shared Access Signature or Access Key, base URL and account name.
```

Comment:
Create an

Reply:
Shall we put them under bullet points?

Reply:
Done

Reply:
> Shall we put them under bullet points?

Yes. It makes sense to put them under bullet points. 
Done in all the similar places